src/components/cursor-controller.js

```diff
@@ -370,10 +376,8 @@ AFRAME.registerComponent("cursor-controller", {
   },
 
   _handleEnterVR: function() {
-    if (AFRAME.utils.device.checkHeadsetConnected()) {
```

Comment:
Why was it necessary to remove this check? In theory it should return true because the polyfill provides a "Cardboard" headset.

Reply:
I'm not sure if thats true, at least it wasn't in the older version of the polyfill. But regardless listening to just enter/exit VR should be sufficient.